### PR TITLE
Branch c

### PR DESCRIPTION
**Pull request by sky-blue2022**
##Key Changes
Created a new feature

##Context of the Changes
Based on customer feedback

### CODE REVIEW

File: git_remote_practice.html [hunk 1]
```diff
@@ -16,6 +16,8 @@ <h1>M3LA</h1>
 <!-- /Branch B-->
 
 <!-- Branch C-->
-N/A
+<h1>C1LB</h1>
+<h1>C2LB</h1>
```
Comment (bloovee) [inline git_remote_practice.html:20]: Found an error. Please fix this line.

**Review by bloovee — CHANGES_REQUESTED** [review]
Overall looking good. Please fix the error before merging it with the master.

File: git_remote_practice.html (hunk 1 above)
Comment (sky-blue2022) [reply git_remote_practice.html]: Fixed the error.

**Comment by sky-blue2022** [comment]
Already fixed the error and pushed new code. Please approve it.

**Review by bloovee — APPROVED** [review]

**Pull request merged**